requirements.txt

```diff
@@ -0,0 +1 @@
+pyFMG
```

Comment:
should set the latest version here so it doesn't randomly break if a newer version of the package gets released